### PR TITLE
[Serverless] Add region, projectID and containerID tags 

### PR DESCRIPTION
**Pull request by maxday**
### What does this PR do?

Ad region + projectID + containerID tags to logs/metrics/traces for CloudRun
Don't tag containerID for metrics (cardinality)
Add tests

### Describe how to test/QA your changes
Screen Shot 2022-05-26 at 4 04 36 PM
Screen Shot 2022-05-26 at 1 53 55 PM
Screen Shot 2022-05-26 at 1 53 30 PM

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: cmd/serverless-init/metadata/metadata.go [hunk 1]
```diff
@@ -6,43 +6,123 @@
...
+		projectIDURL:   fmt.Sprintf("%s%s", defaultBaseURL, defaultProjectID),
+		timeout:        defaultTimeout,
+	}
+}
+
+func GetMetaData(config *config) *Metadata {
+	wg := sync.WaitGroup{}
+	metadata := &Metadata{}
+	wg.Add(3)
+	go func() {
```
Comment (DarcyRaynerDD) [inline cmd/serverless-init/metadata/metadata.go:71]: Just to confirm, there is no way of getting all these values in a single API call?

**Review by DarcyRaynerDD — APPROVED** [review]
Overall LGTM

**Review by vickenty — APPROVED** [review]
Would it make sense to mark `cmd/serverless-init` as owned by the serverless team?

File: cmd/serverless-init/metadata/metadata.go (hunk 1 above)
Comment (maxday) [reply cmd/serverless-init/metadata/metadata.go:71]: that's correct: https://cloud.google.com/run/docs/container-contract#metadata-server

**Comment by maxday** [comment]
> Would it make sense to mark `cmd/serverless-init` as owned by the serverless team?

Definitely! I'll add this to this PR

**Pull request merged**